src/gui/vsAudio.cpp

```diff
@@ -365,7 +370,8 @@ void VsAudio::loadSettings()
     m_inMultiplier  = settings.value(QStringLiteral("InMultiplier"), 1).toFloat();
     m_outMultiplier = settings.value(QStringLiteral("OutMultiplier"), 1).toFloat();
     m_monMultiplier = settings.value(QStringLiteral("MonMultiplier"), 0).toFloat();
-    m_inMuted       = settings.value(QStringLiteral("InMuted"), false).toBool();
+    m_inMuted       = false;
```

Comment:
Are we intentionally keeping this?

Reply:
Yes, I think it's a bug that we were persisting mute state in settings.

In particular it's a problem if you leave a studio while muted, close the app, come back later. When you go to (any of the) AudioSettings you will not be able to make the meters move, because you're muted. So it appears like your audio just isn't working.